invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -116,8 +131,42 @@ export default class DocumentMetadata extends Component {
     return rows;
   }
 
+  _renderPublicAccessRights() {
```

Comment:
Not sure if this has been agreed upon but from #352 we no longer use `_` for class methods.

Reply:
👀 nicely spotted